pages/linux/rusnapshot.md

```diff
@@ -0,0 +1,20 @@
+# rusnapshot
+
+> BTRFS snapshotting utility made in Rust.
```

Comment:
I like prefer the verb `written` to `made` when discussing the software stack.  Cool that it's in Rust! 🚀
```suggestion
> BTRFS snapshotting utility written in Rust.
```

Reply:
I'll fix it right at it.
Also yeah rust is very cool.